### ISSUE TITLE
Action stopped pushing folders prefixed with a dot (ex: .vite)

### ISSUE THREAD

**Issue by GustavoGomez092** [wpengine/github-action-wpe-site-deploy#106]
**Describe the bug**
The GitHub Action stopped pushing folders that are prefixed with a dot (e.g., .vite). These folders are not being included in the push, even though they should be.

**To reproduce**
Create a folder in your project with a dot prefix (e.g., .vite).
Run the action and observe that the folder with the dot prefix is not pushed to WPEngine.

**Expected behavior**
Folders prefixed with a dot, such as .vite, should be pushed to the repository by the GitHub Action as they are part of the project structure.

**Build Output & Screenshots**
No errors or warnings appear in the build output related to these folders being skipped.
Screenshot:
![image](https://github.com/user-attachments/assets/e0e8c81b-d7c3-438b-9d0f-e18efaa74ad6)
![image](https://github.com/user-attachments/assets/9e58a2c4-ae6e-458d-a981-fcd0392fc5fc)

**Comment by apmatthews** [comment]
Hey there @GustavoGomez092,

Excluding files and folders prefixed with a dot is the intended behavior for the action. The exclusion wasn't working properly and a fix was recently shipped in v3.2.6.

If you'd like dot files and folders to be deployed, you can use the `FLAGS` [option](https://github.com/wpengine/github-action-wpe-site-deploy?tab=readme-ov-file#deploy-options). The following configuration uses the [default flags](https://github.com/wpengine/github-action-wpe-site-deploy/blob/7b6fcbc5589c7981993f3ab2aed45f78ab947d90/action.yml#L18C15-L18C30), minus the dot exclusion.

```
name: Deploy to WP Engine
on:
  push:
    branches:
     - main
jobs:
  build:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v4
    - name: GitHub Action Deploy to WP Engine
      uses: wpengine/github-action-wpe-site-deploy@v3
      with:
        WPE_SSHG_KEY_PRIVATE: ${{ secrets.WPE_SSHG_KEY_PRIVATE }}
        WPE_ENV: <your_install_name_here>
        FLAGS: -azvr --inplace
```

**Comment by GustavoGomez092** [comment]
Gotcha! Thank you for letting me know, I was under the wrong impression that the behavior was not intended. Thank you for clarifying. I'll close the issue now.
